VPN UI improvements part 1 - basic copy and UI changes

samsymons opened the pull request:
<!--
Note: This checklist is a reminder of our shared engineering expectations. Feel free to change it, although assigning a GitHub reviewer and the items in bold are required.

⚠️ If you're an external contributor, please file an issue first before working on a PR, as we can't guarantee that we will accept your changes if they haven't been discussed ahead of time. Thanks!
-->

Task/Issue URL: https://app.asana.com/0/414235014887631/1206747946345655/f
Tech Design URL:
CC:

**Description**:

This PR makes basic UI and copy changes to the VPN UI.

<!--
If at any point it isn't actively being worked on/ready for review/otherwise moving forward strongly consider closing it (or not opening it in the first place). If you decide not to close it, use Draft PR while work is still in progress or use `DO NOT MERGE` label to clarify the PRs state and comment with more information.
-->

**Steps to test this PR**:
1. Check that assets and copy have changed per the diff - note that these changes aren't final per the design, this PR simply extracts the more basic noisy changes to keep the other PRs cleaner

<!--
Before submitting a PR, please ensure you have tested the combinations you expect the reviewer to test, then delete configurations you *know* do not need explicit testing.

Using a simulator where a physical device is unavailable is acceptable.
-->

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [ ] iOS 16

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

samsymons committed:
Update copy.

samsymons committed:
Increase VPN toggle view padding.

samsymons committed:
Add status badge.

samsymons committed:
Update VPN assets.

samsymons committed:
Remove now-unused assets.

samsymons committed:
Merge branch 'main' into sam/vpn-ui-improvements

* main:
  Point to BSK hotfix (#2558)
  Prevent redeeming invite codes after the VPN test ends (#2559)
  Updates Kingfisher to 7.11.0 (#2545)
  Makes network path metadata description anonymous (#2544)
  Adds pixels to track main VPN funnels (#2543)
  Cleanup after rolling out autoconsent enabled by default (#2537)
  Add #URL macro (#2540)
  Handle expired entitlement in NetP (#2525)
  Remove isSubscriptionEnabled check when attempting to delete NetP token (#2548)
  Validate correct environment (#2546)
  Remove CGNAT range (#2524)
  Add preliminary support for subscription keychain sharing (#2538)
  fix opening tabs with transitional (#2542)
  Integrate confirm entitlements endpoint (#2541)
  Makes dbSaveBloomFilterError daily and count (#2526)
  16. Subscription: Display "Activation in progress" message (#2535)
  Autofill support for deleting all passwords  (#2497)
  Bump BrowserServicesKit (#2532)
  Release 7.111.0-0 (#2534)
  if dax dialogs are showing then dismiss (#2506)

samsymons committed:
Merge branch 'main' into sam/vpn-ui-improvements

* main:
  Subscriptions: 20 - Subscription Caching (#2569)
  Subscriptions: 19. Error handling and minor updates (#2567)
  Use History on iOS (#2539)
  Release 7.112.0-1 (#2573)
  Report on toggle protections off (#2536)
  Release 7.112.0-0 (#2572)
  Update autoconsent to v10.2.0 (#2554)
  update metadata (#2571)
  Subscription pro pixels (#2531)
  Update SwiftSoup and Kingfisher versions (#2566)
  Move vpnFirstEnabled and networkPathChange out of VPNSettings (#2560)
  Fix VPN view model memory leak (#2570)
  Bump submodules/privacy-reference-tests from `40ce868` to `a603ff9` (#2500)
  18. Subscription Entitlements caching (#2556)
  Release 7.111.0-2 (#2563)
  Revert "Report Apple Ad attribution using pixel (#2510)" (#2562)
  Vanilla browser integration (#2550)
  Fix blank space after URL bar hides (#2549)
  Release 7.111.0-1 (#2561)

samsymons committed:
Merge branch 'main' into sam/vpn-ui-improvements

* main:
  rollback #URL macro (#2585)
  Release 7.112.0-2 (#2579)
  Updates BSK (#2577)

samsymons committed:
Fix unit tests.

samsymons committed:
Merge branch 'main' into sam/vpn-ui-improvements

* main:
  Update wireguard-apple to 1.1.3 (#2598)
  Updates BSK to include latest changes (#2604)
  Fix VPN feedback form description (#2606)
  Fixes the name of a pixel (#2575)
  22. Subscription- Cleanup and minor updates (#2596)
  Release 7.112.0-3 (#2602)
  update embedded file
  Bump autoconsent to 10.3.0 (#2601)
  Roll back CPM post-rollout cleanup (#2599)
  Stub objects for Bookmarks DB (#2593)
  Subscriptions - 21. Manage Billing options to third parties (#2574)
  17. Subscription Review Fixes (#2547)
  bump bsk to fix retain cycle (#2595)
  New log for SKAd 4 integration (#2594)
  NetP x Subscription Clean-up (#2565)
  Update BSK to include latest macOS changes (#2588)

quanganhdo commented on DuckDuckGo/UserText.swift:
```diff
@@ -580,15 +580,16 @@ In addition to the details entered into this form, your app issue report will co
-    static let netPStatusHeaderTitleOff = NSLocalizedString("network.protection.status.header.title.off", value: "Network Protection is Off", comment: "Header title label text for the status view when netP is disconnected")
-    static let netPStatusHeaderTitleOn = NSLocalizedString("network.protection.status.header.title.on", value: "Network Protection is On", comment: "Header title label text for the status view when netP is connected")
-    static let netPStatusHeaderMessage = NSLocalizedString("network.protection.status.header.message", value: "DuckDuckGo's VPN secures all of your device's Internet traffic anytime, anywhere.", comment: "Message label text for the netP status view")
+    static let netPStatusViewTitle = NSLocalizedString("network.protection.status.view.title", value: "VPN", comment: "Title label text for the status view when netP is disconnected")
```
```suggestion
    static let netPStatusViewTitle = NSLocalizedString("network.protection.status.view.title", value: "VPN", comment: "Title label text for the status view when VPN is disconnected")
```

quanganhdo commented on DuckDuckGo/UserText.swift:
```diff
@@ -580,15 +580,16 @@ In addition to the details entered into this form, your app issue report will co
     static let netPStatusConnecting = NSLocalizedString("network.protection.status.connecting", value: "Connecting...", comment: "The label for the NetP VPN when connecting")
     static func netPStatusConnected(since timeLapsedString: String) -> String {
-        let localized = NSLocalizedString("network.protection.status.connected.format", value: "Connected - %@", comment: "The label for when NetP VPN is connected plus the length of time connected as a formatter HH:MM:SS string")
+        let localized = NSLocalizedString("network.protection.status.connected.format", value: "Connected · %@", comment: "The label for when NetP VPN is connected plus the length of time connected as a formatter HH:MM:SS string")
```
```suggestion
        let localized = NSLocalizedString("network.protection.status.connected.format", value: "Connected · %@", comment: "The label for when VPN is connected plus the length of time connected as a formatter HH:MM:SS string")
```

samsymons committed:
VPN UI improvements part 2 - new header animation (#2581)

Task/Issue URL: https://app.asana.com/0/414235014887631/1206747946345656/f
Tech Design URL:
CC:

Description:

This PR adds the new header animation to the iOS UI.

samsymons committed:
Merge branch 'main' into sam/vpn-ui-improvements

# By Anh Do (2) and others
# Via GitHub
* main:
  23. Subscriptions:  - Feature Flags (#2605)
  fix settings maestro text (#2613)
  Web pixels (handlers + pixels) (#2611)
  Handle subscription-related iOS use cases (#2597)
  Remove hardcoded NetP staging endpoint (#2612)
  Add Privacy Manifests to app extensions (#2608)
  Add a VPN metadata section to the debug UI (#2502)
  Use History in Suggestions on iOS (#2552)

# Conflicts:
#	DuckDuckGo.xcodeproj/project.pbxproj
#	DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

samsymons committed:
Update BSK reference.

samsymons committed:
Only show the FAQ section if necessary.

samsymons committed:
Merge branch 'main' into sam/vpn-ui-improvements

* main:
  Add setting to display full URL address (#2587)
  Don't refresh swipe tabs model on background changes (#2610)

samsymons committed:
Update BSK reference.
